server/app.ts

```diff
@@ -19,10 +20,23 @@ import {
 
 // Set up Express and its middleware
 export let app = express();
-app.use(morgan("dev"));
+
 app.use(compression());
 let cookieParserInstance = cookieParser(undefined, COOKIE_OPTIONS);
 app.use(cookieParserInstance);
+morgan.token("sessionid", (request, response) => {
+	const FAILURE_MESSAGE = "Unknown session";
+	if (!request.cookies["connect.sid"]) {
+		return FAILURE_MESSAGE;
+	}
+	let rawID: string = request.cookies["connect.sid"].slice(2);
+	let id = cookieSignature.unsign(rawID, config.secrets.session);
+	if (typeof id === "string") {
+		return id;
+	}
+	return FAILURE_MESSAGE;
+});
+app.use(morgan(":date[iso] :remote-addr :sessionid :method :url :status :response-time ms - :res[content-length]"));
```

Comment:
Can you add back color for status? See https://github.com/expressjs/morgan/blob/master/index.js#L183